packages/renderer/src/api/Renderer.ts

```diff
@@ -0,0 +1,7 @@
+export default interface Renderer {
+  randomPrefix: string;
+
+  renderLayouts(): void;
```

Comment:
Please add some argument like `renderLayoutsRequest: RenderLayoutsRequest` even if it's empty, it's our standard